app/src/main/kotlin/cz/covid19cz/erouska/ui/dashboard/DashboardFragment.kt

```diff
@@ -171,16 +178,18 @@ class DashboardFragment : BaseFragment<FragmentDashboardPlusBinding, DashboardVM
 
         dash_card_positive_test.card_on_content_click =
             View.OnClickListener { viewModel.sendData() }
-        
-        exposure_notification_content.text = AppConfig.encounterWarning
-        exposure_notification_more_info.setOnClickListener { viewModel.showExposureDetail() }
-        exposure_notification_close.setOnClickListener {
-            viewModel.acceptExposure()
-            exposure_notification_container.hide()
-        }
 
         data_notification_close.setOnClickListener { data_notification_container.hide() }
 
+        dash_card_active.setOnClickListener {
+            Analytics.logEvent(requireContext(), ANALYTICS_KEY_PAUSE_APP)
```

Comment:
Log it after the action, not before. (Analytics is secondary, if something goes wrong in analytics, it should not block the action)